templates/admin/event/list.html.twig

```diff
@@ -0,0 +1,199 @@
+{% extends 'admin/base_admin.html.twig' %}
+
+{%block heading %}События{% endblock %}
+
+{% block content %}
+    <div class="container-fluid">
+        <div class="row mb-3" >
+            {% if entities is empty %}
+                Сейчас в базе нет этих данных.
```

Comment:
@daria-sieroshtan 
User never wants to know about DB. He wants to live in illusion that fairies deliver data to his browser from the rainbow.
I suggest something like "Nothing to show"

Reply:
That's an admin-user. Let him know the truth.